Monika After Story/game/updates.rpy

```diff
@@ -255,10 +255,37 @@ label v0_8_4(version="v0_8_4"):
 
         import store.evhand as evhand
         import store.mas_stories as mas_stories
+        import datetime
 
         # update seen status
         updateTopicIDs(version)
 
+        aff_to_grant = 0
+
+        if renpy.seen_label('monika_christmas'):
+            aff_to_grant += 10
+
+        if renpy.seen_label('monika_newyear1'):
+            aff_to_grant += 5
+
+        if renpy.seen_label('monika_valentines_chocolates'):
+            aff_to_grant += 15
+
+        if renpy.seen_label('monika_found'):
+            aff_to_grant += 10
+
+        moni_love = evhand.event_database.get("monika_love", None)
+
+        if moni_love is not None:
+            aff_to_grant += (moni_love.shown_count * 7) / 100
+
+        aff_to_grant += (datetime.datetime.now() - persistent.sessions["first_session"]).days / 3
+
+        if aff_to_grant > 200:
+            aff_to_grant = 200
+
+        persistent._mas_affection["affection"] = aff_to_grant + persistent._mas_affection.get("affection",0)
```

Comment:
shouldnt this use `mas_gainAffecition` so it properly pushes events and stuff

Also this probably will get overwritten by teh affection scrambler so this should run the aff save functoin right after setting